examples/mcmc-DataOrdinal.R

```diff
@@ -1,11 +0,0 @@
-data <- DataOrdinal(
```

Comment:
is it correct that this file is deleted?

Reply:
Yes.  I will restore it in #707.